adding two events and event listener to Fraud and unFraud user

ImedZnd opened the pull request:
adding missing events to Fraud and unFraud user in models and listeners and their tests

ImedZnd committed:
adding two events and event listener tu Fraud and unFraud user

omarbenromdhane reviewed (changes requested):
missing test for DateIsNotValidError
missing error handler for no such message exception for messageSource.getMessage(errorCode, null, Locale.US) for event rest handler
no need for the name for event dto companion object , it is not a builder
it is better to execute a method then map it to the result transformation than to wrap all the method call (example eventService.getAllEvents()
            .map { it.toEventDTO() }
            .let { ok().bodyAndAwait(it) } for getAllEvents for event rest handler
change getAllEventsInRange to accept localdate instead of date 
remove lombok from pom
remove the annotation @repository from ReactiveDatabaseRepository
remove @Autowired from eventHandler function at event configuration file
remove sonar from pom file , set sonar in a pipeline config file
remove r2dbc postgres dependency version from pom
remove hibernate from pom
remove el-impl dependency from pom
remove postgres testcontainer dependency from pom
remove println

ImedZnd committed:
adding toLocalDate to Date
deleting @Autowired in event config
refactoring the event handler and adding test on DateIsNotValid
adding exception handling for no such message exception
change getAllEventsInRange to accept LocalDate
deleting @Repository annotation from repository
adding test for DateIsNotValidError
removing useless dependecies from POM
adding swagger ui dependency in POM
adding HistoryRouterInfo.kt annotation
adding @OpenAPI in main class
changing the sonar config to the pipeline file

ImedZnd committed:
removing the companion object name

ImedZnd committed:
change sonar config

ImedZnd committed:
change sonar config

ImedZnd committed:
change sonar config

ImedZnd committed:
change sonar config

ImedZnd committed:
change sonar config
